charts/openshift-metering/templates/presto/presto-coordinator-config.yaml

```
@@ -59,6 +59,18 @@ data:
 {{- end }}
 
   jvm.config: |
+{{- if .Values.networking.useGlobalProxyNetworking.true }}
+{{- if .Values.presto.spec.config.tls.enabled }}
+    -Dhttp.proxyHost=networking.useGlobalProxyNetworking.proxy.config.http_proxy.hostname
+    -Dhttp.proxyPort=networking.useGlobalProxyNetworking.proxy.config.http_proxy.port
+    -Dhttp.proxyUser=networking.useGlobalProxyNetworking.proxy.config.http_proxy.username
+    -Dhttp.proxyPassword=networking.useGlobalProxyNetworking.proxy.config.http_proxy.password
+{{ else }}
+    -Dhttps.proxyHost=networking.useGlobalProxyNetworking.proxy.config.https_proxy.hostname
+    -Dhttps.proxyPort=networking.useGlobalProxyNetworking.proxy.config.https_proxy.port
+    -Dhttps.proxyUser=networking.useGlobalProxyNetworking.proxy.config.https_proxy.username
+    -Dhttps.proxyPassword=networking.useGlobalProxyNetworking.proxy.config.https_proxy.password
```

(this applies to the entire block of code, this is an invalid path)
```suggestion
    -Dhttps.proxyPassword=networking.proxy.config.https_proxy.password
```